libs/app-updater/ui/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export * from "./lib/checking-for-update-modal"
```

Comment:
⚪ Consistent naming with `update-` prefix
```suggestion
export * from "./lib/update-checking-modal"
```